eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -54,6 +54,18 @@
       "RunAggregateReports": false,
       "TestGoals": "surefire:test",
       "TestOptions": ""
+    },
+    {
+      "Agent": {
+        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+      },
+      "JavaTestVersion": "1.20",
+      "AZURE_TEST_HTTP_CLIENTS": "netty",
+      "TestFromSource": false,
+      "RunAggregateReports": false,
+      "TestGoals": "surefire:test",
+      "TestOptions": "",
+      "IsLatestNonLtsJdk": true
```

Comment:
This one could be combined into the above entry via `"IsLatestNonLtsJdk": [false, true]`

Reply:
The above entry as in the Java 11 entry? Would the JavaTestVersion then need to be `[1.11, 1.20]`?